app/src/main/res/values/strings.xml

```diff
@@ -558,9 +557,7 @@
     <!-- Updating webview message -->
     <string name="Webview.updating">Updating...</string>
     <!-- Welcome screen text. (?) will be replaced with the help icon users should look for. -->
-    <string name="Welcome.body">Ravencoin Wallet has changed its name to RVN, with a brand new look and some new features.\n\nIf you need help, look for the (?) in the top right of most screens.</string>
```

Comment:
@ROSHiix Are these strings just not used anymore? Why leave the comment text about the strings that are no longer there?

Reply:
This is not used anymore, I must remove it from code first.